docs/maven-tutorial.md

```diff
@@ -0,0 +1,218 @@
+	"metadata": {
+		"name": "nexus-pv",
+		"labels": {
+			"type": "local"
+		}
+	},
+	"spec": {
+		"hostPath": {
+			"path": "/tmp/nexus-pv"
+		},
+		"accessModes": [
+			"ReadWriteOnce"
+		],
+		"capacity": {
+			"storage": "1Gi"
+		},
+		"persistentVolumeReclaimPolicy": "Retain"
+	}
+}
+```
+
+Import that file into OpenShift.
+```bash
+oc create -f pv.json
+```
+
+Log back in to your user, then add a Persistent Volume Claim to the Nexus
+Deployment Config.  Using `oc cluster up`, the default user is 'developer'
+```bash
+oc login -u developer
+oc volumes dc/nexus --remove --confirm
+oc volumes dc/nexus --add \
+	--name 'nexus-pvc' \
+	--type 'pvc' \
+	--mount-path '/sonatype-work/' \
+	--claim-name 'nexus-pv' \
+	--claim-size '1G'
+```
+
+This removes the previous volume for the Deployment Config, and adds a claim
+for 1 gigabyte of storage mounted at `/sonatype-work`, which is where our
+dependencies will be stored.  Due to the change in config, the Nexus image will
+be redeployed automatically.
+
+Verify that Nexus is running by refreshing the Nexus page in your browser.
+
+## Modifying the Builder
+
+The next steps will be modifying our builder image to use this repository.
+The rest of the tutorial will be editing
+[this sample repository](https://github.com/bparees/openshift-jee-sample.git),
+but these changes should work for any project.
+
+Create a directory in your project called 'wildfly-builder'.  Inside this
+direcotry, create a file in your project called `settings.xml` with the
+following content:
+```xml
+<settings>
+  <mirrors>
+    <mirror>
+      <!--This sends everything else to /public -->
+      <id>nexus</id>
+      <mirrorOf>*</mirrorOf>
+      <url>http://nexus:8081/content/groups/public</url>
+    </mirror>
+  </mirrors>
+  <profiles>
+    <profile>
+      <id>nexus</id>
+      <!--Enable snapshots for the built in central repo to direct -->
+      <!--all requests to nexus via the mirror -->
+      <repositories>
+        <repository>
+          <id>central</id>
+          <url>http://central</url>
+          <releases><enabled>true</enabled></releases>
+          <snapshots><enabled>true</enabled></snapshots>
+        </repository>
+      </repositories>
+     <pluginRepositories>
+        <pluginRepository>
+          <id>central</id>
+          <url>http://central</url>
+          <releases><enabled>true</enabled></releases>
+          <snapshots><enabled>true</enabled></snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+  </profiles>
+  <activeProfiles>
+    <!--make the profile active all the time -->
+    <activeProfile>nexus</activeProfile>
+  </activeProfiles>
+</settings>
+```
+
+This is the crucial file for the effort.  We're telling Maven to use only the
+repository at nexus:8081, which is our proxy.  More information on DNS
+resolution in OpenShift can be found
+[here](https://docs.openshift.com/enterprise/3.1/architecture/additional_concepts/networking.html)
```

Comment:
Why we're using link for Enterpise version and not to Origin (https://docs.openshift.org/latest/architecture/additional_concepts/networking.html) ?